Duplicati/Library/Utility/Utility.cs

```diff
@@ -974,16 +974,16 @@ public static string LocateFileInSystemPath(string filename)
         /// <summary>
         /// The path to the users home directory
         /// </summary>
-        private static readonly string HOME_PATH = Environment.GetFolderPath(IsClientLinux ? Environment.SpecialFolder.Personal : Environment.SpecialFolder.UserProfile);
+        public static readonly string HOME_PATH = Environment.GetFolderPath(IsClientLinux ? Environment.SpecialFolder.Personal : Environment.SpecialFolder.UserProfile);
 
         /// <summary>
-        /// Expands environment variables, including the tilde character
+        /// Expands environment variables.
         /// </summary>
         /// <returns>The expanded string.</returns>
         /// <param name="str">The string to expand.</param>
         public static string ExpandEnvironmentVariables(string str)

```

Comment:
This method is no longer necessary, as we can just call the built-in method.  I decided to leave this refactoring for a separate pull request so that we don't pollute the diff in this pull request.

Reply:
Actually, it seems that `ExpandEnvironmentVariables` will only expand Windows-style environment variables (e.g., it will expand `%HOME%` but not `$HOME`).  Do we need to better support Linux and MacOS here?

Reply:
Yes, I have a comment about that in the code. Until now it has only used the Windows-style environment variable expansion (that is how .Net does) but it is simple to do the Linux style expansion as well.

Should we just drop the old Windows-style environment variable expansion on non-Windows? I am not sure.... I think it may break something for current users expecting the expansion, but new users will be confused if it does not work as all the other stuff on their system....

Reply:
I'm not sure what the best approach is.  Is the main purpose here to expand environment variables specified in command-line options and filters?  If so, one option is to only support (and document) the Windows-style syntax.  Unless there's a way to support both on non-Windows environments if we want to avoid breaking current functionality?

I'll create a separate issue regarding this matter so we can continue the discussion there.

Reply:
Created issue #2838.